draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -448,6 +448,9 @@ A Merkle Tree certification authority is defined by the following values:
 `validity_window_size`:
 : An integer describing the maximum number of unexpired batches which may exist at a time. This value is determined from `lifetime` and `batch_duration` by `lifetime / batch_duration`.
 
+`http_server`:
+: A servername together with optional port and path prefix, under which the CA publishes, as described in {{publishing}}. For example: `example.com/ca`.
```

Comment:
If we go with the `tlog` approach (#103), we probably want to make this the same as the submission prefix/origin line from https://github.com/C2SP/C2SP/blob/main/static-ct-api.md#checkpoints.